pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -227,6 +227,22 @@ func doUpdateContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus)
 				blobErrorEntities = append(blobErrorEntities, &types.ErrorEntity{EntityID: blob.Sha256, EntityType: types.ErrorEntityContentBlob})
 
 				leftToProcess = true
+				if blob.RetryCount < blobDownloadMaxRetries {
```

Comment:
Do we set blob.HasError when it is the verifier service which detects that the sha doesn't match?